packages/datagrid/src/constants/avro-type.constant.js

```diff
@@ -1 +1,12 @@
-export default ['boolean', 'int', 'long', 'float', 'double', 'bytes', 'string'];
+export const AVRO_TYPES = ['boolean', 'int', 'long', 'float', 'double', 'bytes', 'string'];
+export const LOGICAL_TYPES = ['timestamp-millis'];
```

Comment:
Is the only logical type for the dates ?

Reply:
yes